src/features/retrospective/ui/RetrospectiveSummary.tsx

```
@@ -0,0 +1,99 @@
+import { useMemo } from 'react';
```

ESLint

[error] 'useMemo' is defined but never used.

(@typescript-eslint/no-unused-vars)